Makefile

```diff
@@ -42,7 +42,10 @@ run-appengine-staging:
 
 run-api-server:
 	test $(SERVICE_ACCOUNT) || (echo "SERVICE_ACCOUNT variable not set"; exit 1)
-	cd gcp/api && pipenv sync && GOOGLE_CLOUD_PROJECT=oss-vdb pipenv run python test_server.py $(SERVICE_ACCOUNT)
+	# Required for ESP container to be able to read the credential, the configured alpine image doesn't respect the volume mapping args.
+	chmod 644 $(SERVICE_ACCOUNT)
```

Comment:
Yeah this is a good point, I tried quite a few variations before I got here but this syntax:
`-v "$HOME/.config/gcloud:/gcp/config:ro"` does not seem to work with this particular container image. 

I was trying to avoid modifying the image, but that is the only alternative I can think of?  We would create our own image that adds the dependencies required to do that and/or adds the credentials themselves to the image, which would mean copies of those creds would live in docker images locally accessible to anybody in the docker group.

I'll see if I can amend the image without having to copy the creds in as that seems the least bad option